parsers/sort-by/sort-by.spec.ts

```diff
@@ -4,13 +4,13 @@ import { Token } from '../../types';
 import libs from '../global-libs';
 
 describe('sort-by', () => {
-  it('Get tokens - apply parsers', async () => {
+  it('Sorts tokens alphabetically by their name', async () => {
```

Comment:
Rebased the PR to remove this change that already done in another PR

Reply:
Done ✅

Reply:
This change shouldn't be in this PR since it concern another parser